README.md

```diff
@@ -21,6 +21,8 @@ A `.github/stale.yml` file is required to enable the plugin. The file can be emp
 daysUntilStale: 60
 # Number of days of inactivity before a stale Issue or Pull Request is closed
 daysUntilClose: 7
+# Number of days of inactivity before a stale Issue or Pull Request is locked
```

Comment:
I don't think locking is about inactivity, but the days after the stale issue is closed. What about something like this?

> Number of days after a stale Issue or Pull Request has been closed to lock it

Reply:
Yeah I like that more.